src/server/upgrade.rs

```diff
@@ -89,6 +89,24 @@ fn interpret_options(options: &Upgrade) -> anyhow::Result<ToDo> {
 
 pub fn upgrade(options: &Upgrade) -> anyhow::Result<()> {
     let todo = interpret_options(&options)?;
+    if let ToDo::InstanceUpgrade(name, version) = &todo {
+        let project_dirs = destroy::find_project_dirs(name)?;
+        if !project_dirs.is_empty() {
+            destroy::print_instance_in_use_warning(name, &project_dirs);
+            eprintln!("Go to each project directory and run:");
```

Comment:
Multiple projects per instance is an exception, not the rule, so let's specialize the message to the common use case.  Also, for better DX, let's use `--project-dir` instead of asking the user to `cd` manually (probably only if the current directory isn't already that of the project).